cranelift/codegen/src/isa/x86/unwind/winx64.rs

```diff
@@ -183,41 +136,42 @@ pub(crate) fn create_unwind_info(
 
     assert!(found_end);
 
-    if saved_fpr {
-        if static_frame_allocation_size > 240 && saved_fpr {
-            warn!("stack frame is too large ({} bytes) to use with Windows x64 SEH when preserving FPRs. \
-                This is a Cranelift implementation limit, see \
-                https://github.com/bytecodealliance/wasmtime/issues/1475",
-                static_frame_allocation_size);
-            return Err(CodegenError::ImplLimitExceeded);
+    let mut frame_register_offset = 0;
+    if xmm_save_count > 0 {
```

Comment:
This should actually check if a frame pointer is being used, even though we currently always use one.

If a frame pointer is not used, then the offsets for the FPRs saves should not be altered as they already have the correct offset from RSP.

I'll quickly fix this.